packages/rs-dapi-client/src/lib.rs

```diff
@@ -71,9 +71,19 @@ impl<T: transport::TransportRequest + Send> DapiRequest for T {
     }
 }
 
-/// Allows to flag the transport error variant how tolerant we are of it and whether we can
-/// try to do a request again.
+/// Returns true if the operation can be retried.
```

Comment:
I think having the third state None doesn't provide additional benefits but makes it more complex.

Reply:
done